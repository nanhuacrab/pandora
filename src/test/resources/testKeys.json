[{
  "dimensionsSize": [1],
  "testCases": [{
    "dimensionsValues": [
      ["中国"]
    ],
    "expected": "testKeys.1_1.expected"
  }]
},{
  "dimensionsSize": [3],
  "testCases": [{
    "dimensionsValues": [
      ["中国","日本","韩国"]
    ],
    "expected": "testKeys.3_1.expected"
  }]
},{
  "dimensionsSize": [2,3],
  "testCases": [{
    "dimensionsValues": [
      ["浙江","上海"],
      ["舟山","杭州","上海"]
    ],
    "expected": "testKeys.2-3_1.expected"
  }]
},{
  "dimensionsSize": [1,2,3],
  "testCases": [{
    "dimensionsValues": [
      ["中国"],
      ["浙江","上海"],
      ["舟山","杭州","上海"]
    ],
    "expected": "testKeys.1-2-3_1.expected"
  }]
}]