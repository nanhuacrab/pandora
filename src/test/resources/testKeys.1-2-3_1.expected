["中国,浙江,舟山,","中国,浙江,杭州,","中国,浙江,上海,","中国,上海,舟山,","中国,上海,杭州,","中国,上海,上海,"]